resources/css/flamma-tokens.css

```
@@ -70,9 +70,9 @@
     border-radius: 50%;
     cursor: pointer;
 
-    background: var(--orange-primary);
+    background: var(--brand-primary);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)